README.md

```diff
@@ -58,9 +64,10 @@ go install github.com/manuelarte/embeddedstructfieldcheck@latest
 And then use it as:
 
 ```bash
-embeddedstructfieldcheck [--fix]
+embeddedstructfieldcheck [-forbid-mutex=true|false] [--fix]
 ```
 
+- `forbid-mutex`: `true|false` (default `false`) Checks that `sync.Mutex` is not used as an embedded field.
 - `fix`: Fix the case when there is no space between the embedded fields and the regular fields.
```

Comment:
🎨 minor stylistic idea

`fix` also supports using `-fix=true/false`

What about this?

````suggestion
embeddedstructfieldcheck [-forbid-mutex] [--fix]
```

- `forbid-mutex`: `true|false` (default `false`) Checks that `sync.Mutex` is not used as an embedded field.
- `fix`: `true|false` (default `false`) Fix the case when there is no space between the embedded fields and the regular fields.
````

And then maybe thus would be clearer

````suggestion
embeddedstructfieldcheck [-forbid-mutex] [--fix]
```

- `forbid-mutex`: `true|false` (default `false`)
   Checks that `sync.Mutex` is not used as an embedded field.
- `fix`: `true|false` (default `false`)
   Fix the case when there is no space between the embedded fields and the regular fields.
````